public_html/style.css

```diff
@@ -0,0 +1,255 @@
+/* Common style css */
+:root {
+
+    /* color */
```

Comment:
The hex values seem slightly off here. I annotated each color entity below with my suggestions. Please check with Johan. Maybe you got these values from his Figma file?

Reply:
Yeah I got all color from figma file